src/ingest_validation_tools/table-schemas/level-2/mixif.yaml

```diff
@@ -14,7 +14,7 @@ fields:
   -
     name: analyte_class
     constraints:
-      required: False
+      required: proteins
```

Comment:
@cebriggs7135 I believe that this field name is supposed to be "enum" rather than "required".